medusa/server/api/v2/scene_exception.py

```diff
@@ -16,20 +16,67 @@ def get_last_updates():
     return last_updates
 
 
-class SceneExceptionOperationHandler(BaseRequestHandler):
-    """Scene Exception operation request handler."""
+class SceneExceptionTypeHandler(BaseRequestHandler):
+    """Scene Exception type request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionTypeHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'GET')
+
+    def get(self, exception_type):
+        if exception_type is not None and exception_type not in ['medusa', 'xem', 'anidb']:
```

Comment:
I think changing to naming exception is fine. But then should be changed in a big bang. Db, api, code, comments, wiki. So don't think this Pr is really a good location to discuss that.